### PR TITLE
Minor syntax correction.

### PR DESCRIPTION
**Pull request by mbasset**
Fix an issue that prevents latest cookbook from functioning correctly. The tests are not passing but don't appear to be related to the change.

### CODE REVIEW

File: providers/server_monitor.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@ def update_newrelic_alert_policy_linux(alert_policy_id)
     end
 
     only_if do
-      node['newrelic']['api_key'].!empty?
+      !node['newrelic']['api_key'].empty?
```
Comment (rmldsky) [inline providers/server_monitor.rb]: One can use `not_if` block instead and drop the `!`. Would be much cleaner this way.

**Comment by rmldsky** [comment]
Would love to get this merged. Wanted to submit similar PR, but found this one. All cookbook's versions `>= 2.21.0` suffer from this. Which makes adding servers to NewRelic impossible and fails `chef-client` run.

I've also added in-diff comment about the fix.

File: providers/server_monitor.rb (hunk 1 above)
Comment (mbasset) [inline providers/server_monitor.rb]: Good idea. I have updated the PR.

**Comment by mbasset** [comment]
@djoos What needs to be done to get this into the master branch?

**Comment by rmldsky** [comment]
@djoos great it got merged, thou master had already some version in `metadata.rb` shouldn't this be bumped as well? Not sure what release flow you have. Maybe it is a good time to bump up cookbook version and tag?

**Comment by djoos** [comment]
We make use of semver on the cookbook - I'll bump up by patch or include into a bigger release.
However, the latest build seems broken, so nothing will be shipping until fixed!

**Comment by djoos** [comment]
@mbasset - thanks for your contribution!

**Comment by djoos** [comment]
Ok, those failures are because of an open PR - shipping this particular PR in patch version 2.22.4.

Thanks!

**Pull request merged**